src/Inviqa/Worldpay/Api/Request/AuthoriseRequestTreeBuilder.php

```diff
@@ -107,6 +107,7 @@ public function buildHcgAdditionalData()
             new Param(new Name('xField3'), new ParamValue('')),
             new Param(new Name('xField4'), new ParamValue($this->parameters['checkoutMethod'])),
             new Param(new Name('xField5'), new ParamValue($this->parameters['orderSource'])),
+            new Param(new Name('xField6'), new ParamValue($this->parameters['paymentSubtype'] ?? '')),
```

Comment:
Are you sure that the `paymentSubtype` key exist at this point? I don't see any usage of it in the source code.
It should be set in `\Inviqa\Worldpay\Api\Request\AuthorizeRequestFactoryApplePay::$defaultApplePayParameters` and `\Inviqa\Worldpay\Api\Request\AuthorizeRequestFactoryGooglePay::$defaultGooglePayParameters` arrays in my opinion.

Reply:
It will come from request params, please check here https://github.com/inviqa/reiss/pull/5652

Reply:
I see. What's the benefit of passing this data from the application that uses the library? What would happen if a new application wants to use this Worldpay library for Apple Pay or Google Pay?

Reply:
@timcwebster Could you please clarify. Because I'm not so aware about this Worldpay API library.

Reply:
I think if you add them as empty values in the default params in the two classes mentioned above then it will be fine.

Reply:
@elvetemedve  I have added default value for this param to 
`\Inviqa\Worldpay\Api\Request\AuthorizeRequestFactory::DEFAULT_PARAMETERS`